Website/htdocs/mpmanager/app/Services/PluginsService.php

```diff
@@ -53,4 +43,12 @@ public function getByMpmPluginId($mpmPluginId)
     {
         return $this->plugin->newQuery()->where('mpm_plugin_id', $mpmPluginId)->first();
     }
+
+    public function isPluginActive(int $SPARK_METER): bool
+    {
+        return $this->plugin->newQuery()
+            ->where('mpm_plugin_id', '=', $SPARK_METER)
+            ->exists();
+    }
```

Comment:
Can you rename the function argument? `$SPARK_METER` is misleading, as this can be called with any `plugin_id`, right?

Reply:
sure, the function implementation was created by the IDE based on the call from the spark meter package.